Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -687,6 +700,10 @@ TextureCache::TCacheEntryBase* TextureCache::Load(const u32 stage)
 		}
 	}
 
+	// If we are recording a FifoLog, keep track of what memory we read.
+	if (g_bRecordFifoData && !from_tmem)
+		FifoRecorder::GetInstance().WriteMemory(address, texture_size + additional_mips_size, MemoryUpdate::TEXTURE_MAP);
```

Comment:
This could be moved to the top of this function. It will slow down while the fifo is recorded, but so it won't catch hashing issues,